Signal/src/ViewControllers/ConversationView/ConversationViewItem.m

```diff
@@ -323,8 +323,12 @@ - (DisplayableText *)displayableTextForInteractionId:(NSString *)interactionId
             displayText = @"";
         }
 
-        displayableText =
-            [[DisplayableText alloc] initWithFullText:fullText displayText:displayText isTextTruncated:isTextTruncated];
+        NSNumber *_Nullable jumbomojiCount = [DisplayableText jumbomojiCountTo:fullText];
```

Comment:
Are we using this class method elsewhere? 
Do we ever init a DisplayableText without a jumbomojiCount?

nit: If the answer to both is "no" then I'd argue this belongs in the DisplayableText inititializer.

Reply:
Done.